### DISCUSSION TITLE
组织名称的翻译与意义？

### DISCUSSION THREAD

**Discussion by WorldHim** [Meloong-Git/PCL#3526] [提问] [answered]
`Hex Dragon` 这个词应该如何翻译，还是作为专有名词保留？
看到很多人在提到机器人时都将这个翻译为 `六角龙`，这显然不够信达雅。

以下是个人见解：
`Hex` 可代表 16 进制，算是与计算机相关的词汇，翻译为 `六角` 不太合理

**Comment by WForst-Breeze** [comment, marked as answer]
个人认为以专有名词处理（即称呼其原始名称）最佳，除非龙猫真的去取了一个中文名。毕竟 “六角龙” 事实上是社区交流中的机翻产物，如果真的要进行翻译感觉交流上会怪怪的。就像我们讨论时绝大部分时候会说 “Issue” 而不是 “议题”，会说 “Pull Request” 而不是 “拉取请求” 一样，对于模棱两可的称谓最好的处理方式便是保持原样。

当然，归根到底社区用户怎么去称呼我们肯定也是管不到的，毕竟这是讨论的自由，就像玩家们管 Mojang 叫麻酱一样。

**Comment by Deep-Dark-Forest** [comment]
~~最亮的信标称其为…三角龙工作室~~
个人认为没有必要去规范，因为这就是个习惯问题，你去规范了反而一些已经看惯了edge自动翻译的人会觉得很难受

**Comment by burningtnt** [comment]
十六龙

**Comment by iPodtouch6557** [comment]
信达雅在某些场合会造成翻译的不实用性，且社区已经默认达成共识，故得不更改

**Comment by LTCatt** [comment]
事实：我也还没考虑过中文名要叫啥

**Reply by allMagicNB** [reply]
那 Hex 是六角的意思还是十六进制的意思（

**Reply by SqrtTan** [reply]
会不会是用什么狗屁不通句子生成器/GPT搞的

**Reply by iPodtouch6557** [reply]
六角更习惯，Hex(agon)嘛
十六进制有些别扭
